Makefile

```diff
@@ -269,3 +269,14 @@ schema-check: ## Check that func.yaml schema is up-to-date
 	(echo "\n\nFunction config schema 'schema/func_yaml-schema.json' is obsolete, please run 'make schema-generate'.\n\n"; rm -rf schema/func_yaml-schema-previous.json; exit 1)
 	rm -rf schema/func_yaml-schema-previous.json
 
+######################
+##@ Hack scripting
+######################
+
+.PHONY: regenerate-kn-components
+regenerate-kn-components: # Regenerate hack/component-versions(.json/.sh) locally
```

Comment:
We have a make target `help` which prints an index of targets.  Entries are added by the double-hash prefix on the comment `## Regenerate hack/component-versions`.  Perhaps worth adding the extra hash so that the help index shows this new target